app/views/layouts/application.html.erb

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
`Ruby on Rails Tutorial Sample App`
tìm hiểu về I18n
sử dụng I18n cho toàn bộ text của các file html

và sử dụng `link_to` thay vì tag `a` của html nhé